deltann/server/core/model/delta_model.go

```diff
@@ -72,7 +81,7 @@ func DeltaModelRun(valueInputs interface{}) (string, error) {
 		deltaPtr := C.CString(valueInputs.(string))
 		defer C.free(unsafe.Pointer(deltaPtr))
 		ins.ptr = unsafe.Pointer(deltaPtr)
-		ins.size = C.int(len(valueInputs.(string)))
+		ins.size = C.int(len(valueInputs.(string)) + 1)
```

Comment:
putting `+1` outter.

Reply:
this is unique to string and is already in the case string